proto/buf/alpha/registry/v1alpha1/download.proto

```diff
@@ -21,20 +21,38 @@ import "buf/alpha/module/v1alpha1/module.proto";
 // DownloadService is the download service.
 service DownloadService {
   // Download downloads.
+  // Soon to be deprecated.
```

Comment:
We should state it is soon to be deprecated in favor of DownloadManifest.

Reply:
Sure, we could expand on _why_ we're not deprecating it _yet_.